packages/amplify-cli/src/domain/amplify-toolkit.ts

```diff
@@ -258,6 +259,13 @@ export class AmplifyToolkit {
       this._showResourceTable || require(path.join(this._amplifyHelpersDirPath, 'resource-status')).showResourceTable;
     return this._showResourceTable;
   }
+
+  get showStatusTable(): any {
+    this._showStatusTable =
+    this._showStatusTable || require(path.join(this._amplifyHelpersDirPath, 'resource-status')).showStatusTable;
```

Comment:
Can this be changed to await import?

Reply:
This will require conversion of showStatusTable function into async . This is a different behavior compared to other functions in this class. For now I feel it's better to keep it as require and update it once for all functions in this class as part of performance update.